gsitemap-cron.php

```diff
@@ -31,27 +31,30 @@
 include(dirname(__FILE__).'/../../config/config.inc.php');
 include(dirname(__FILE__).'/../../init.php');
 /* Check to security tocken */
-if (substr(Tools::encrypt('gsitemap/cron'), 0, 10) != Tools::getValue('token') || !Module::isInstalled('gsitemap'))
-	die('Bad token');
+if ('cli' != php_sapi_name()) {

```

Comment:
Hello,

In some server configuration, when PHP is running in cgi mode per exemple php_sapi_name() will returns cli. You should use the Tools::isPHPCLI() method.

Reply:
Fixed

Reply:
Is the fix correct?